actions/delegator/setup-token/src/main.ts

```diff
@@ -76,6 +76,7 @@ async function run(): Promise<void> {
         sha: process.env.GITHUB_SHA,
         workflow_ref: process.env.GITHUB_WORKFLOW_REF,
         workflow_sha: process.env.GITHUB_WORKFLOW_SHA,
+        workflow: process.env.GITHUB_WORKFLOW,
```

Comment:
note: I believe I removed this intentionally since it's not a very exact field and we can uniquely identify the workflow from the `workflow_ref` and `workflow_sha`.

Reply:
Posting a follow-up, thanks for the feedback. I agree let's remove this - it's usually just he workflow name  anyway